workshops/facial_expression_recognizer/README.md

```diff
@@ -0,0 +1,155 @@
+---
+name: 'Facial Expression Recognizer'
+description: 'Build FER with Python'
+author: '@YashKalbande'
+img: 'https://cloud-ki1smi0m8.vercel.app/0finaloutput-min.gif'
+---
+# Introduction
+In our daily life, we knowingly or unknowingly carry out different types of Facial Expression. These movements convey the emotional state of an individual.
+We can judge the mood and mental state of the next person by his Facial Expression. In the early Twentieth century, Ekman and Friesen defined `six` basic emotions.
+This expression does not change with cultures, they are universal. Six Facial Expressions are:-
+- **Happiness** 😃
+- **Sadness**😔
+- **Fear**😨
+- **Anger**😤
+- **Suprise**😲
+- **Disgust**😖
+
+![Happiness image](https://cloud-kmaqeh4qc.vercel.app/happy1.gif)
+![Sadness image](https://cloud-kmaqeh4qc.vercel.app/sad1.gif) 
+![Fear image](https://cloud-kmaqeh4qc.vercel.app/fear1.gif) 
+
+![anger image](https://cloud-kmaqeh4qc.vercel.app/angry1.gif)
+![suprise image](https://cloud-kmaqeh4qc.vercel.app/suprise1.gif) 
+![disgust image](https://cloud-kmaqeh4qc.vercel.app/disgust1.gif)
+
+Here's the [**demo**](https://repl.it/@YashKalbande/facialexpressionrecongnizer#main.py) of what we'll be making. You can see both the final code and test it out on this Repl environment.
+
+![Final Code Output](https://cloud-ki1smi0m8.vercel.app/0finaloutput-min.gif)
+
+In this workshop, I'll share how to build a Facial Expression Recognizer using the `FER` library from python.
+
+## FER Library
+Facial Expression Recognition Library is developed by [`Justin Shenk`](https://pypi.org/user/jshenk). This Library requires `OpenCV>=3.2` and `Tensorflow>=1.7.0` dependencies installed in the system. Faces are detected using OpenCV's Haar Cascade classifier. For more information and the Source code of FER Library, you can visit [FER's GitHub page here.](https://github.com/justinshenk/fer)
+
+## Setting up our code!
+
+For this workshop, we're going to be using an online code editor called [Repl.it](https://repl.it/site/about), or you can use your favorite code editor. It can be installed through pip:`$ pip install fer`. If you can using Repl so just import the `FER` library and it will be installed with all its dependencies.
+
+1. Head over to [Repl.it/languages/Python3](https://repl.it/languages/python3)
```

Comment:
Add a NOTE somewhere here saying the readers to make sure they are using python 3 and above. I tried doing the workshop in python 2 and was confused because I didn't know about this.